### ISSUE TITLE
LUKS+ZFS

### ISSUE THREAD

**Issue by karasu** [Antergos/Cnchi#513]
Enable encryption+zfs in zfs.py

**Comment by fourstone** [comment]
Hi,

problem I see with current CNCHI is that one can specify to use encrypted ZFS in the Installation Dialog and no warning appears telling you that encryption with ZFS is currently not possible.

Frankly I consider it a BUG therefore in the current form.

Installing manually, LUKS with ZFS works just fine needing systemd encrypt-sd booting when using multiple partitions underneath ZFS for e.g. a RAID1 configuration.

Question obviously is, how many people actually want that and of these people how many would really use the installer way rather than going pacstrap. (BTW pacstrap antergos-base does NOT work -> will open an issue for that)

kind regards, Martin

**Comment by karasu** [comment]
Yes, as you have guessed, this type of setup is far from being a "normal" user setup. This is not among our target (Cnchi's target) for sure.

When we added zfs, we planned to add encrypting with it, but then we discarded the option. That's why there must still be an option to select it, but it does not work for sure.

I'll check that dialog to remove the option.

Thanks for the warning! (I'll keep this open as a reminder, and I'll close it when the check has been made).

**Comment by genofire** [comment]
I hope you really take the LUKS encryption and not the weak zfs encryption
